### DISCUSSION TITLE
Distinction between anime video and old-school illustration upscaling

### DISCUSSION THREAD

**Discussion by TomLucidor** [OpenModelDB/open-model-database#324] [General]
Just a bit of an annoyance from a tag search perspective since most are for old anime upscaling which is known for lessened complexity (for most but not all scenes) compared to the illustration of the same era, which can be very intricate. Maybe there is a way of making the  distinction?

**Comment by RunDevelopment** [comment]
We can make tags as fine-grained as we want, so if you have a good way of differentiating between different types of anime models, feel free to open an issue. 

FYI: Our tagging system also supports implications between tags (if tag X is set, then tag Y is guaranteed to be set), so tags don't have to be mutually exclusive. E.g. we could have "Old-school Anime", "Modern Anime", and "Anime" tags where both "Old" and "Modern" imply the "Anime" tag.
If your tags have more complex relationships, then describe the relationship in detail, and we might be able to make the tagging system powerful enough to handle the relationship.

**Reply by TomLucidor** [reply]
I think the major distinction is that Illustration is more similar to one another (western or cartoon vs anime-style illustration); while anime screencaps or anime video upscaling have their own set of issues similar to video frame fixing. How I conceptualize this:
|  | Illustration | Video |
|--|--|--|
| Compression and Noise | JPEG artifact reduction (noisy compression) | Denoising (recording error), Debanding (low bit rate) |
| Structural Errors | Dedither (pixelation), Dehalftone (manga color) | Dehalo, Derainbow, Debluring |
